src/blocksparse/blocksparsetensor.jl

```diff
@@ -668,6 +668,30 @@ function permutedims!!(
   #end
 end
 
+# <fermions>
+function scale_blocks!(T, compute_fac::Function=(b) -> 1) where {N}
+  # intentionally left blank
```

Comment:
I generally have used the convention that in-place functions would have a `return T` here.